frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -480,12 +479,13 @@ end
 
 function ReaderDictionary:_genDownloadDictionariesMenu()
     local downloadable_dicts = require("ui/data/dictionaries")
+    local isoLanguage = require("ui/isolanguage")
```

Comment:
(You can keep its Uppercase Initial `I` :) It's not the fact that it's now no longer a module level require that should make it lose its uppercase - we have many `local ConfirmBox = require("ui/widget/confirmbox")` - and `local confirmbox = ConfirmBox:new{}` in functions - it's rather if it is a class or not, `downloadable_dicts` is probably just a table, but your `IsoLanguage`, having methods, probably deserves being called `local IsoLanguage`.)

Reply:
Okay, I'll change it back
Shouldn't that socket stuff lower down follow that same logic then? That's what I looked at, I figured I should follow that